content/realtime/channels/channel-parameters/deltas.textile

```diff
@@ -72,57 +84,30 @@ TBD
 
-When subscribing without an Ably library, then the channel @delta@ param must be specified using a "qualified channel name":/realtime/channels/channel-parameters. In the case of [SSE](https://www.ably.io/documentation/sse), it is also possible to specify channel params as regular query params on the connection URL.
+When subscribing without an Ably library, then the channel @delta@ parameter must be specified using a "qualified channel name":/realtime/channels/channel-parameters/overview. In the case of SSE, it is also possible to specify channel parameters as regular query parameters on the connection URL.
 
-Some transports provide raw message payloads - that is, the content of the @data@ attribute of a @Message@ - without the accompanying metadata. That means that the recipient of the message does not have access to the @extras@ or @encoding@ attributes of the message that would ordinarily be used to decode delta message payloads. Examples of such transports are [MQTT](https://www.ably.io/documentation/mqtt), and [SSE](https://www.ably.io/documentation/sse) in non-enveloped mode. In order to assist applications that use these transports, the @vcdiff@ decoder libraries can check for the @vcdiff@ magic number at the start of the message payload as an inexact method of determining whether or not the message is a regular message or a delta. Note that, in order to rely on that check, you need to know that that magic number will not be present in any valid (uncompressed) message in your app. No valid JSON value, for example, will match the @vcdiff@ header check, so it is safe to perform this sniffing on JSON message payloads.
+Some transports provide raw message payloads - that is, the content of the @data@ attribute of a @Message@ - without the accompanying metadata. That means that the recipient of the message does not have access to the @extras@ or @encoding@ attributes of the message that would ordinarily be used to decode delta message payloads. 
 
-h4(#delta-example-sse). Delta example with SSE
+Examples of such transports are MQTT, and SSE in non-enveloped mode. In order to assist applications that use these transports, the @vcdiff@ decoder libraries can check for the @vcdiff@ magic number at the start of the message payload as an inexact method of determining whether or not the message is a regular message or a delta. Note that, in order to rely on that check, you need to know that that magic number will not be present in any valid (uncompressed) message in your app. No valid JSON value, for example, will match the @vcdiff@ header check, so it is safe to perform this sniffing on JSON message payloads.
```

Comment:
@paddybyers can I clarify the use of the term 'magic number' here. Is it merely a colloquialism or that what it's actually referred to as?

Reply:
It's generally referred to as a magic number, yes. See https://en.wikipedia.org/wiki/Magic_number_(programming) (second definition).

Reply:
great thanks